### DISCUSSION TITLE
Are there plans for CodeIgniter 3.1.14?

### DISCUSSION THREAD

**Discussion by El-Tommy** [bcit-ci/CodeIgniter#6265] [Q&A]
I know that CodeIgniter 4 is already there, with support for PHP 8.X, but our entire platform was built with CodeIgniter 3, coming from CodeIgniter 2.

But rewriting everything for CodeIgniter 4 will take another year, taking into account rewriting and approval.

So I ask if there are still plans for CodeIgniter 3.1.14?
Or is version 3 already definitively closed with no plans for continuation?

**Comment by Noaber** [comment]
Hopefully, see also https://github.com/bcit-ci/CodeIgniter/pull/6173

**Comment by kenjis** [comment]
There is no official plans for CI3 in CodeIgniter Foundation.
But @gxgpet has been working on #6173 since Nov 6, 2022.

> First, the foundation hasn't decided anything about CI3 - it never needed to, never had to, and to my knowledge was never intended to deal with CI3 at all. I've been at the helm since forever and this is all on me.
https://github.com/bcit-ci/CodeIgniter/issues/6012#issuecomment-775048076
